src/api/moviesApi.ts

```diff
@@ -0,0 +1,8 @@
+export const moviesApis = {
+    GetMovies: (movieGenre?: string, sortBy = 'release_date') => {
+        const filterBy = `&filter=${movieGenre}`
+        const sortType = `&sortBy=${sortBy}&sortOrder=desc`
+
+        return `${process.env.REACT_APP_API}/movies?limit=9&offset=0${filterBy}${sortType}`
+    },
```

Comment:
For query params use [query-string](https://www.npmjs.com/package/query-string) or similar lib.